test/e2e/config/e2e_conf.yaml

```diff
@@ -57,12 +84,22 @@ providers:
 - name: metal3
   type: InfrastructureProvider
   versions:
-  - name: v1.1.0
-    value: "file://${HOME}/.cluster-api/overrides/infrastructure-metal3/v1.1.0/infrastructure-components.yaml"
+  - name: v0.5.5
+    value: "https://github.com/metal3-io/cluster-api-provider-metal3/releases/download/v0.5.5/infrastructure-components.yaml"
     type: "url"
+    contract: v1alpha4
+    files:
+    - sourcePath: "../data/infrastructure-metal3/metadata.yaml"
```

Comment:
Please note that we are using same metadata for both CAPM3 v1alpha5 and v1beta1 
@lentzi90 @furkatgofurov7

Reply:
@mboukhalfa why is that, it was using its own metadata for each API version?

Reply:
Well I am convinced that first it not shared since it is specific for the provider and also one file can include all releases and it is clearly mapping it to the contract. 
Basically we were including all the previous release for the current contract I mean v1alpha4 was including v1alpha3 release so it would harm if it got the newer also like v1beta1. Although capi uses multiple file I will check the reason behind that